### DISCUSSION TITLE
Is it possible to Build a In Process Background Task for UWP

### DISCUSSION THREAD

**Discussion by Agredo** [shinyorg/shiny#848] [Q&A]
Reading through your doku and reading you jobs sample project I am wondering if it is possible to use a in process bacground task (no need for a Background.UWP Project) for UWP. https://docs.microsoft.com/en-us/windows/uwp/launch-resume/create-and-register-an-inproc-background-task

Thank you :)

**Comment by aritchie** [comment]
Theoretically it should, but I have not tried it.  UWP isn't something I work with much at all.

**Reply by Agredo** [reply]
Okay Thank you I will check this out. Thank you :)

**Reply by Agredo** [reply]
Okay it should be possible but I dont know where I can set RunInProc to true: 
![image](https://user-images.githubusercontent.com/16531090/146072518-c771e3a5-e428-4922-b3a8-355dfdea6ea0.png)

Here is the link to the file. 
https://github.com/shinyorg/shiny/blob/96da140390445634ce48f038ed0099b8e961595a/src/Shiny.Core/Platforms/Uwp/UwpPlatform.cs#L123

Thank you

**Reply by aritchie** [reply]
At the very beginning of your app.  I'm sorry, but you really are on your own for this one.  UWP has been a pain and I don't use it professionally at all.
